tests/src/JIT/SIMD/AbsGeneric.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace VectorMathTests
+{
+    class Program
+    {
+
+
+
+        static int Main(string[] args)
+        {
+			short[] arr = new short[] {-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5};
+            var a = new System.Numerics.Vector<short>(arr);
+            a = System.Numerics.Vector.Abs(a);
+            if (a[0] != 5)
+            {
+                return 0;
+            }
+            var b = System.Numerics.Vector<int>.One;
+            b = System.Numerics.Vector.Abs(b);
+            if (b[3] != 1)
+            {
+                return 0;
+            }
+            var c = new System.Numerics.Vector<long>(-11);
+            c = System.Numerics.Vector.Abs(c);
+            if (c[1] != 11)
+            {
+                return 0;
+            }
+
+            var d = new System.Numerics.Vector<double>(-100.0);
+            d = System.Numerics.Vector.Abs(d);
+            if (d[0] != 100)
```

Comment:
In the particular case of `abs` I think that equality is warranted, I wouldn't expect `abs(-100.0)` to be something like 100.000000000001.